### PR TITLE
Feature/cc 1885

### PR DESCRIPTION
**Pull request by rDr4g0n**
Update UI to use RAMLimit instead of RAMCommitment per:
https://jira.zenoss.com/browse/CC-1829
https://jira.zenoss.com/browse/CC-1885

### CODE REVIEW

File: acceptance/ui/features/data/default/hosts.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
       "hostID": 4982,
       "memory": 10737418240,
       "memoryGB": "10.00 GB",
-      "ramGB": "5.34 GB",
+      "ramGB": "5737418240",
```
Comment (tgjonestx) [inline acceptance/ui/features/data/default/hosts.json]: Is this really the value we show to the user?

File: web/ui/src/common/miscUtils.js [hunk 2]
```diff
@@ -183,6 +192,37 @@
                 }, 0);
 
                 return childCount;
+            },
+
+            validateRAMLimit(limitStr, max=Infinity){
```
Comment (tgjonestx) [inline web/ui/src/common/miscUtils.js]: Can you add some unit-tests for this method?

File: web/ui/src/common/miscUtils.js [hunk 3]
```diff
@@ -183,6 +192,37 @@
...
+
+                var isPercent = (limitStr.endsWith("%"));
+
+                // if this is a percent, ensure its between 1 and 100
+                if(isPercent){
+                    let val = +limitStr.slice(0, -1);
+                    if(val > 100){
+                        return "RAM Limit cannot exceed 100%";
```
Comment (tgjonestx) [inline web/ui/src/common/miscUtils.js]: Assuming these are error messages to be displayed to the user, do they need to be localized?

File: acceptance/ui/features/data/default/hosts.json (hunk 1 above)
Comment (rDr4g0n) [inline acceptance/ui/features/data/default/hosts.json]: We are making it a point to keep the unit the user enters, so if they choose to use bytes, then they will always see bytes. They can go back in and change it to megs or gigs or whatever they prefer.

File: web/ui/src/common/miscUtils.js (hunk 3 above)
Comment (rDr4g0n) [inline web/ui/src/common/miscUtils.js]: I haven't been localizing phrases like these, mostly just sticking to nouns and verbs. In the future we can make a pass at localization (and automation of localization using google translate api)

**Pull request merged**